Task3/src/ThirdClass.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+public class ThirdClass {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the first number");
+        double num1 = sc.nextDouble();
+        System.out.println("Choose on of the possible operations: +, -, *, /, %");
+        char operator = sc.next().charAt(0);
```

Comment:
add operations validation here, no needs  to ask second digital if operation is incorrect

Enter the first number
4
Choose on of the possible operations: +, -, *, /, %
;
Enter the second number
4
Your input is not correct.